bluesky/simple_scans.py

```diff
@@ -66,8 +67,8 @@ class _BundledScan:
 
     def __init__(self):
         # subs and sub_factories can be set individually per instance
-        self.subs = self.default_subs
-        self.sub_factories = self.default_sub_factories
+        self.subs = list(self.default_subs)
```

Comment:
Sigh. We had this conversation really carefully because I know it runs again your general taste. At the time you said it made sense to accept users' "Do what I mean!" code at this layer. There is no ambiguity.